### PR TITLE
Add arch to lev ratio and helper methods

### PR DESCRIPTION
**Pull request by yotamdaniel**
- Add Arch to lev ratio paramater in ParamStore + setter and getter
- Add 4 helper methods to get how much lev is allowed based on burned Arch 
- General code clean up and making sure tests dont always fail on github

### CODE REVIEW

File: contracts/ParameterStore.sol [hunk 1]
```diff
@@ -41,19 +42,23 @@ contract ParameterStore is AccessController {
...
-    function changeRebaseFeeRate(uint256 _newRebaseFeeRate) external {
+    function changeRebaseFeeRate(uint256 newRebaseFeeRate) external {
         // rebaseFeeRate must be a number between 1 and 99 (in 18 decimal)
-        require(_newRebaseFeeRate < (100 ether) && _newRebaseFeeRate > (0 ether), "New rebase fee rate out of range");
-        _rebaseFeeRate = _newRebaseFeeRate;
+        require(newRebaseFeeRate < (100 ether) && newRebaseFeeRate > (0 ether), "New rebase fee rate out of range");
+        _rebaseFeeRate = newRebaseFeeRate;
+    }
+
+    function changeArchToLevRation(uint256 newArchToLevRation) external {
```
Comment (yotamdaniel) [inline contracts/ParameterStore.sol:60]: change to ratio

File: integrationTests/zIntegrationTests.ts [hunk 2]
```diff
@@ -176,9 +176,9 @@ describe("Test suit for getting leverage", function () {
     });
 
     it("Should have created a single position and assign it to user", async function () {
-        printPositionState(r, positionId);
-        printPoolState(r.curveLvUSDPool);
-        printMiscInfo(r, user);
+        // printPositionState(r, positionId);
```
Comment (yotamdaniel) [inline integrationTests/zIntegrationTests.ts:179]: remove comment

File: integrationTests/zIntegrationTests.ts [hunk 3]
```diff
@@ -176,9 +176,9 @@ describe("Test suit for getting leverage", function () {
...
+        // printPoolState(r.curveLvUSDPool);
```
Comment (yotamdaniel) [inline integrationTests/zIntegrationTests.ts:180]: remove comment

File: integrationTests/zIntegrationTests.ts [hunk 4]
```diff
@@ -215,8 +215,8 @@ describe("test suit for rebase events", function () {
         await r.vault.takeRebaseFees();
     });
     it("Should update treasury with rebase fees", async function () {
-        printPositionState(r, positionId);
-        printMiscInfo(r, user);
+        // printPositionState(r, positionId);
```
Comment (yotamdaniel) [inline integrationTests/zIntegrationTests.ts:218]: Remove comment

File: test/ParameterStore.ts [hunk 5]
```diff
@@ -153,6 +153,16 @@ describe("ParameterStore test suit", async function () {
                     ethers.utils.parseUnits("368.559"),
                 );
             });
+            it("Should return lower leverage allowance if not enough ArchToken", async function () {
```
Comment (yotamdaniel) [inline test/ParameterStore.ts:156]: Set origination fee to current default so test always runs

**Review by jasonstockman — APPROVED** [review]

**Pull request merged**